src/index.ts

```diff
@@ -1,12 +1,14 @@
-import { Fraction, transitiveOrderbook } from '@gnosis.pm/dex-contracts'
+import { Fraction} from '@gnosis.pm/dex-contracts'
```

Comment:
ubernit: formatting is a bit off.

Reply:
How do I auto format in javascript? I tried to use vscode but it changed formatting for lines I didn't touch too and it didn't seem to ever split long lines.

Reply:
Maybe @fleupold can give you some pointers, I have no idea

Reply:
The .eslintrc.js should have the formatting definitions. VSCode does split long lines for me (maybe you need to install the eslint package for that to work).